packages/flutter/test/rendering/dynamic_intrinsics_test.dart

```diff
@@ -83,4 +83,42 @@ void main() {
     pumpFrame();
     expect(root.size, equals(inner.size));
   });
+
+  group('When RenderObject.debugCheckingIntrinsics is true', () {
+    setUp(() => RenderObject.debugCheckingIntrinsics = true);
+    tearDown(() => RenderObject.debugCheckingIntrinsics = false);
```

Comment:
can we do this inside the test? it's much clearer what is going on if we don't use `test`'s various features. (For example, the timing of `testWidgets`'s invariants check and `tearDown` isn't obvious.) That would also let you get rid of `group`.

Reply:
Sure, I replaced this with a try-finally to reset `RenderObject.debugCheckingIntrinsics` and removed the `group`.